camunda-sdk-java/java-common/src/main/java/io/camunda/common/http/DefaultHttpClient.java

```diff
@@ -116,12 +119,13 @@ public <T> String getXml(Class<T> selector, Long key) {
     String url = host + basePath + retrievePath(selector) + "/" + key + "/xml";
     HttpGet httpGet = new HttpGet(url);
     httpGet.addHeader(retrieveToken(selector));
-    String xml = null;
+    String xml;
     try {
       CloseableHttpResponse response = httpClient.execute(httpGet);
-      xml = new String(Java8Utils.readAllBytes(response.getEntity().getContent()), StandardCharsets.UTF_8);
+      xml = (String) parseAndRetry(response, selector);
```

Comment:
As discussed this will break as it will try to parse the XML as JSON.

Reply:
Addressed in https://github.com/camunda-community-hub/spring-zeebe/pull/588/commits/bc36a55b3e21845ca2d13816e30eb145cd98f9ee